mmcv/transforms/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from .builder import TRANSFORMS
+from .formatting import (ImageToTensor, RandomFlip, RandomResize, ToTensor,
+                         to_tensor)
```

Comment:
`RandomFlip` and `RandomResize` may be put in processing.py

Reply:
Thank you for your valuable coments. I forgot to mv them to processing.py.